presto-main/src/test/java/io/prestosql/operator/TestDynamicFilterSourceOperator.java

```diff
@@ -103,7 +113,8 @@ private OperatorFactory createOperatorFactory(DynamicFilterSourceOperator.Channe
                 this::consumePredicate,
                 Arrays.stream(buildChannels).collect(toList()),
                 getDynamicFilteringMaxPerDriverRowCount(TEST_SESSION),
-                getDynamicFilteringMaxPerDriverSize(TEST_SESSION));
```

Comment:
let's make `maxPerDriverRowCount` and `maxPerDriverSize` params of `createOperatorFactory` (with default override maybe), so that in case we increase limits in the future, the tests won't run longer

Reply:
Done